tests/testing/make_sdists.py

```diff
@@ -6,12 +6,13 @@
 def random_string():
     """return a short suffix that shouldn't collide with any subsequent calls"""
     import os
-    import base64
+    import string
+    import random
 
-    return '.'.join((
-        str(os.getpid()),
-        base64.urlsafe_b64encode(os.urandom(3)),
-    ))
+    return '{}.{}'.format(
+        os.getpid(),
+        ''.join(random.choice(string.ascii_lowercase) for x in range(8)),
```

Comment:
The problem is `base64.urlsafe_b64encode` wants a string, not bytes, on Python 3. You'd have to use `base64.encodebytes` which isn't Python 2-compatible. I figured switching to this was better than using version detection.